melos.yaml

```diff
@@ -10,6 +10,8 @@ command:
     linkToCommits: true
     # Additionally build a changelog at the root of the workspace.
     workspaceChangelog: true
+    hooks:
+      preCommit: melos run update-version
```

Comment:
With this, running [`melos version`](https://melos.invertase.dev/commands/version) will update all the `version.dart` file with the proper version values.